src/util.ts

```diff
@@ -43,28 +43,39 @@ export function toUnixPath(filePath: string): string {
 
 export function convertStringtoSymbolKind(kind: string): SymbolKind {
 	switch (kind) {
-		case 'file': return SymbolKind.File;
 		case 'module': return SymbolKind.Module;
-		case 'namespace': return SymbolKind.Namespace;
-		case 'package': return SymbolKind.Package;
 		case 'class': return SymbolKind.Class;
+		case 'local class': return SymbolKind.Class;
+		case 'interface': return SymbolKind.Interface;
+		case 'enum': return SymbolKind.Enum;
+		case 'enum member': return SymbolKind.Constant;
+		case 'var': return SymbolKind.Variable;
+		case 'local var': return SymbolKind.Variable;
+		case 'function': return SymbolKind.Function;
+		case 'local function': return SymbolKind.Function;
 		case 'method': return SymbolKind.Method;
+		case 'getter': return SymbolKind.Method;
+		case 'setter': return SymbolKind.Method;
 		case 'property': return SymbolKind.Property;
-		case 'field': return SymbolKind.Field;
 		case 'constructor': return SymbolKind.Constructor;
-		case 'enum': return SymbolKind.Enum;
-		case 'interface': return SymbolKind.Interface;
-		case 'function': return SymbolKind.Function;
-		case 'variable': return SymbolKind.Variable;
-		case 'constant': return SymbolKind.Constant;
-		case 'string': return SymbolKind.String;
-		case 'number': return SymbolKind.Number;
-		case 'boolean': return SymbolKind.Boolean;
-		case 'array': return SymbolKind.Array;
-		case 'array': return SymbolKind.Array;
-		case 'sourcefile': return SymbolKind.File;
+		case 'parameter': return SymbolKind.Variable;
+		case 'type parameter': return SymbolKind.Variable;
 		case 'alias': return SymbolKind.Variable;
-		default: return SymbolKind.String;
+		case 'let': return SymbolKind.Variable;
+		case 'const': return SymbolKind.Constant;
+		case 'JSX attribute': return SymbolKind.Property;
+		// case 'script'
+		// case 'keyword'
+		// case 'type'
+		// case 'call'
+		// case 'index'
+		// case 'construct'
+		// case 'primitive type'
+		// case 'label'
+		// case 'directory'
+		// case 'external module name'
+		// case 'external module name'
```

Comment:
That's okay, but they should be mapped to _something_, so lets not comment out the cases